src/components/Router/index.js

```diff
@@ -137,25 +145,34 @@ class MainRouter extends Component {
     super(props)
     this.state = {
       crossDeviceInitialStep: null,
+      i18n: this.initializeI18n()
```

Comment:
I could not get `setOptions` to update the language at runtime

Reply:
Oh, I didn't consider that. Is it supposed to trigger a rerender with the new options? In that case, I could implement it as part of the another ticket given that `useModal` breaks when changed at runtime.

Reply:
Done